hr_employee_second_lastname/tests/test_hr_employee_second_lastname.py

```diff
@@ -0,0 +1,173 @@
+from odoo import exceptions
+from odoo.tests.common import TransactionCase
+from odoo.tools import submap
+
+
+class TestEmployeeLastnames(TransactionCase):
```

Comment:
You forgot to update lastnames -> second lastname here.

Reply:
I just updated this, could you review again, please?